### PR TITLE
Add GitHub.com Available Seats Policy

### PR DESCRIPTION
**Pull request by bryankaraffa**
### Description

Adds the GitHub.com Available Seats Policy Template

### Issues Resolved

N/A

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/github/available_seats/available_seats.pt [hunk 1]
```diff
@@ -0,0 +1,105 @@
+name "GitHub.com Available Seats Report"
+rs_pt_ver 20180301
+short_description "Gets the number of available seats for a licensed GitHub.com Org and creates an incident if they are out of the policy range."
```
Comment (srpomeroy) [inline cost/github/available_seats/available_seats.pt]: Need a `long_description`. It's contents should be: "Version: 1.0"

Comment (srpomeroy) [inline cost/github/available_seats/available_seats.pt]: The short description should have a link to the README file. Checkout another policy for an example.

File: cost/github/available_seats/available_seats.pt [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    summary_template "GitHub Orgs with low available seats"
+    detail_template <<-EOS
+# GitHub Orgs with Low Seats
+| Org Name | Available Seats | Used Seats | Total Seats | 
+| --------- | --------------- | --------- | ------------ |
+{{ range data -}}
+| {{.org}} | {{.available_seats}} | {{.used_seats}} | {{.total_seats}} |
+{{ end -}}
+
+#### Policy Min Seat Count: {{parameters.param_min_seats}}
```
Comment (srpomeroy) [inline cost/github/available_seats/available_seats.pt]: Recommend adding a footer to the detail template that provides information about the CMP Porject the policy is applied to. Example: https://github.com/rightscale/policy_templates/blob/master/cost/scheduled_reports/scheduled_report.pt#L591-L592

File: cost/github/available_seats/available_seats.pt [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    summary_template "GitHub Orgs with too many available seats"
+    detail_template <<-EOS
+# GitHub Orgs with Too Many Available Seats
+| Org Name | Available Seats | Used Seats | Total Seats | 
+| --------- | --------------- | --------- | ------------ |
+{{ range data -}}
+| {{.org}} | {{.available_seats}} | {{.used_seats}} | {{.total_seats}} |
+{{ end -}}
+
+#### Policy Max Seat Count: {{parameters.param_max_seats}}
```
Comment (srpomeroy) [inline cost/github/available_seats/available_seats.pt]: Recommend adding a footer to the detail template that provides information about the CMP Porject the policy is applied to. Example: https://github.com/rightscale/policy_templates/blob/master/cost/scheduled_reports/scheduled_report.pt#L591-L592

File: cost/github/available_seats/available_seats.pt (hunk 1 above)
Comment (srpomeroy) [reply cost/github/available_seats/available_seats.pt]: Example: https://github.com/rightscale/policy_templates/blob/master/cost/volumes/unattached_volumes/uav_policy.pt#L4

Comment (bryankaraffa) [reply cost/github/available_seats/available_seats.pt]: Fixed in 117facc

Comment (bryankaraffa) [reply cost/github/available_seats/available_seats.pt]: Fixed in 4dd1b8d

File: cost/github/available_seats/available_seats.pt (hunk 2 above)
Comment (bryankaraffa) [reply cost/github/available_seats/available_seats.pt]: Fixed in 634b68b

File: cost/github/available_seats/available_seats.pt (hunk 3 above)
Comment (bryankaraffa) [reply cost/github/available_seats/available_seats.pt]: Fixed in 634b68b

**Comment by ronmessana-concerto** [comment]
https://giphy.com/gifs/oGO1MPNUVbbk4/html5

**Pull request merged**